src/doc/rustdoc/src/what-is-rustdoc.md

```diff
@@ -93,6 +93,26 @@ passes `-L`, a flag that helps rustdoc find the dependencies
 your code relies on. If our project used dependencies, we'd get
 documentation for them as well!
 
+## Outer and inner documentation
+The `///` syntax is used to document the item present after it. That's why it is called
+an outer documentation.
+There is another syntax: `//!`, which is used to document the item it is present inside.
+It is called an inner documentation.
```

Comment:
there are newlines after this sentence and the previous that should not be here.